From a406dddaf921d1eea10aafcb399291f3a78e1200 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Wed, 11 Dec 2019 09:40:08 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable CONFIG_REGULATOR_TPS65132

Enable TPS65132 regulator driver to support dual output power
supply used in LCD panels for RK3399 M series tablet board.

Change-Id: I8f1c03cc8c99423216da79efdaed23d96ab4c0c2
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index eb641fcbc24d..7fdc30fa473d 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -525,6 +525,7 @@ CONFIG_REGULATOR_LP8752=y
 CONFIG_REGULATOR_MP8865=y
 CONFIG_REGULATOR_PWM=y
 CONFIG_REGULATOR_RK808=y
+CONFIG_REGULATOR_TPS65132=y
 CONFIG_REGULATOR_XZ3216=y
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
-- 
2.35.3

